__DEFINES/silicon.dm

```diff
@@ -45,11 +42,14 @@ var/global/list/mommi_modules = list(
 	GRAVEKEEPER_MOMMI		= /obj/item/weapon/robot_module/mommi/cogspider
 	)
 
+//Global list of all Cyborg/MoMMI modules. If you add a new list and forget to add it to this one i'll fucking break your neck.
+var/global/list/all_robot_modules = nanotrasen_robot_modules + syndicate_robot_modules + special_robot_modules + mommi_modules
+
 /proc/getAvailableRobotModules()
-	var/list/modules = nanotrasen_robot_modules
+	var/list/pickable_modules = nanotrasen_robot_modules
 	if(security_level != SEC_LEVEL_RED)
-		modules.Cut(COMBAT_MODULE)
-	return modules
+		pop(pickable_modules) //Removes the last module in the list, the combat one.
```

Comment:
Surely nobody will add something at the end of the list, ever

Reply:
This really, really shouldn't have been done this way